### DISCUSSION TITLE
Runtime Logs get POST request body

### DISCUSSION THREAD

**Discussion by cisco0x60** [vercel/community#4020] [Feature Question]
### Question

Is it possible to look at the POST request body from the project **Logs** tab?
It would be a handy feature to have in my opinion.

Moreover, what does the **Request ID** property is used for if I cannot investigate deeper the request itself?

Thank you

**Comment by kapitanluffy** [comment]
Any updates on this?

I also have the same question. What is the point of the Request ID?

**Comment by wingedfox** [comment]
Please add the post body into the logs, I have bunch of the functions failing within 10 second limit and have no idea how to reproduce this.

**Comment by bondansebastian** [comment]
This is much needed

**Comment by abbas-paradox** [comment]
We need this too. 🙏

**Comment by versecafe** [comment]
@francestu96 Request ID is used to link together all logs pertaining to a given request, here's an example

<img width="1196" alt="Screenshot 2024-06-17 at 3 39 44 PM" src="https://github.com/vercel/community/assets/147033096/01205f18-aa3d-42cb-8010-d257d35fc923">
